config/webpack.config.dev.js

```diff
@@ -119,8 +119,18 @@ module.exports = {
       // allow it implicitly so we also enable it.
       {
         test: /\.json$/,
+        exclude: /\/manifest.json$/,
         loader: 'json'
       },
+      // A special case for manifest.json to place it into build root directory.
+      {
+        test: /\/manifest.json$/,
+        include: [paths.appSrc],
```

Comment:
@gaearon sure, can you please explain why should it load differently from the top level in the code and differently from the template? I've tried some configurations and can't get it to load differently even though it's in the top level (by top level you mean `template` dir right?